widget_bandsplot/bandsplot.py

```diff
@@ -112,7 +116,8 @@ def __init__(
                     )
                 )
 
-                temp_dos["dos"][i]["x"] = tx[index].tolist()
-                temp_dos["dos"][i]["y"] = ty[index].tolist()
+                ymin.append(min(ty[index]))
+                ymax.append(max(ty[index]))
 
+            self.dos_range = [min(ymin) * 1.05, max(ymax) * 1.05]
```

Comment:
I think this is the code that caused the drawback you described. Can you add a comment in the code so in the future we can understand it ?

Reply:
Yes, this line is fixing the range of the density acoording to the maximum and miminum values. I added a comment as you suggested.